manager/testdata/new-blueprints/read-transform-blueprint.yaml

```diff
@@ -0,0 +1,95 @@
+apiVersion: app.m4d.ibm.com/v1alpha2
+kind: Blueprint
+metadata:
+  name: read-and-transform
+  labels:
+spec:
+  cluster: thegreendragon
+  appSelector: # Selector of the application that uses this workload
+    clusterName: thegreendragon
+    workloadSelector:
+      matchLabels:
+        app: demoapp
+  steps:
```

Comment:
I would suggest changing steps to modules, since this is a list with no particular order.  I.e. blueprint doesn't include data path order, just the modules that need to be deployed/configured